courses/migrations/0007_course_course_id.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9 on 2015-12-14 20:18
+from __future__ import unicode_literals
+
+import datetime
+from django.db import migrations, models
+from django.utils.timezone import utc
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('courses', '0006_author_name_optional'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='course',
+            name='course_id',
+            field=models.TextField(default=datetime.datetime(2015, 12, 14, 20, 18, 3, 515705, tzinfo=utc), help_text='course locator from edx', unique=True),
```

Comment:
The default here doesn't make sense for a text field and the migration fails due to the unique constraint if you have existing data. I'm fine with clearing the database for this PR but please make a special note of it in the PR message and let @bdero know so we can handle this on the deploy servers